airbyte-integrations/connectors/source-confluence/source_confluence/manifest.yaml

```diff
@@ -152,8 +162,19 @@ spec:
         description: 'Please follow the Jira confluence for generating an API token: <a href="https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/">generating an API token</a>.'
         airbyte_secret: true
         order: 1
-      domain_name:
-        title: "Domain name"
-        description: "Your Confluence domain name"
+      base_url:
+        title: "Base URL"
+        description: "Your Confluence base URL"
```

Comment:
In theory you don't need to change this variable as you need to continue to specify the server endpoint.

Reply:
Changed back.